src/include/platform/internal/GenericPlatformManagerImpl_FreeRTOS.ipp

```diff
@@ -275,6 +275,17 @@ void GenericPlatformManagerImpl_FreeRTOS<ImplClass>::PostEventFromISR(const Chip
 template <class ImplClass>
 void GenericPlatformManagerImpl_FreeRTOS<ImplClass>::_Shutdown(void)
 {
+    GenericPlatformManagerImpl<ImplClass>::_Shutdown();
+
```

Comment:
Why not try to stop the event loop and join the thread? Doing vQueueDelete if anything is pending could crash

Reply:
The expectation is that the event loop is already stopped here, by calling PlatformManager's `StopEventLoopTask` earlier.

But yes, joining the thread here seems like a good idea.  What I don't know is how to do that, even after poking through https://www.freertos.org/a00112.html and related things for a while.  Any hints?

Reply:
Updated the PR to not delete things here.